examples/draft-0-10-0/rich/rich.html

```diff
@@ -43,14 +43,13 @@
           this.focus = () => this.refs.editor.focus();
           this.onChange = (editorState) => this.setState({editorState});
 
-          this.handleKeyCommand = (command) => this._handleKeyCommand(command);
+          this.handleKeyCommand = this._handleKeyCommand.bind(this);
```

Comment:
Yeah, this is the only one who doesn't actually, just checked. Both link and media do as well. I'll update this to make the other 3 functions use bind as well.